spec/features/donuts/02_user_add_new_donut_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+
+feature "user adds a new donuts", %{
+   As a user
+   I want to be able to add a new donut
+   So that a user can add reviews
+  } do
+  # ACCEPTANCE CRITERIA:
+  # [ X ] Root is Donut#index
+  # [ ] See Add New Donut Link on Root
+  # [ ] On donut#new there should be a Form
+  # [ ] Submitting correct form takes us to donut#show
+  # [ ] Submitting incorrect renders the form with errors
+  before(:each) do
+    visit root_path
+  end
+  scenario "User sees Add New Donut link on root " do
```

Comment:
Whitespace is free! Put a line break here to separate each test from anything else